add-dockerfiles.sh

```diff
@@ -33,6 +33,7 @@ scala2.12-java11-python3-r-ubuntu
 scala2.12-java11-python3-ubuntu
 scala2.12-java11-r-ubuntu
 scala2.12-java11-ubuntu
+scala2.13-java11-ubuntu
```

Comment:
You only add the scala one, do you think python3/r/all should also be added in sometime? (Yep, just a question, IMO we can add scala2.13 now, and add others on demand in future)

Reply:
I didn't see any other bin file that was compiled with Scala 2.13 in the [Apache Spark Archives](https://archive.apache.org/dist/spark/spark-3.4.1/). So I thought to hold off on trying the others. But now that I see they are built on top of the initial image (not from the Dockerfile.template, but from the r-python.template), I can try to add support in this PR.